src/child/actions/sidebar.js

```diff
@@ -8,3 +9,26 @@ export const selectSearch = createActionCreator(() => ({
 export const selectFavourites = createActionCreator(() => ({
     type: ACTION_TYPES.FAV_CLICKED
 }));
+
+const windowReopened = createActionCreator((reopenWindowName) => ({
+    type: ACTION_TYPES.REOPEN_WINDOW,
+    reopenWindowName
+}));
+
+const openWindowError = createActionCreator(() => ({
```

Comment:
Worth passing the `windowName` too? Although looks like this action isn't handled at the moment?